omero-web/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM openmicroscopy/omero-ssh-systemd
+ARG ZIPURL=omego
+ENV ZIPURL ${ZIPURL}
+
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -e "s/Defaults    requiretty.*/ #Defaults    requiretty/g" /etc/sudoers
```

Comment:
:+1: you might as well just delete the whole line:
`sed -i -re 's/Defaults\s+requiretty.*//' /etc/sudoers`